src/frontend/src/store/index.ts

```diff
@@ -2,21 +2,46 @@ import axios from "axios";
 import Vue from "vue";
 import Vuex from "vuex";
 import VueToast from 'vue-toast-notification';
+import { sha256 } from "js-sha256";
 import 'vue-toast-notification/dist/theme-sugar.css';
+import * as rs from 'jsrsasign';
 
 type Transaction = {
   to: string;
   from: string;
   amount: number;
+  id: string;
+  signature: string;
 };
 
-function uuidv4() {
-  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
-    const r = Math.random() * 16 | 0, v = c == 'x' ? r : (r & 0x3 | 0x8);
-    return v.toString(16);
-  });
+// Transaction Signing
+const genKeyPair = (): string[] => {
+  const keyPair = rs.KEYUTIL.generateKeypair("RSA", 1024);
+  console.log(keyPair)
+  return [rs.KEYUTIL.getPEM(keyPair.prvKeyObj, "PKCS1PRV"), rs.KEYUTIL.getPEM(keyPair.pubKeyObj)];
 }
 
+const keyPair = genKeyPair();
+const privateKey = keyPair[0];
+const publicKey = keyPair[1];
+
+const getTransactionId = (transaction: Transaction): string => {
+  return sha256(
+    transaction.to + transaction.from + transaction.amount
+  ).toString();
+};
+
+const sign = (transaction: Transaction, privateKey: string): string => {
+  const dataToSign = transaction.id;
+  const sig = new rs.KJUR.crypto.Signature({alg: 'SHA256withRSA'})
+
+  sig.init(privateKey);
+  sig.updateString(dataToSign);
+
+  return sig.sign();
+};
+// end 
```

Comment:
Can we please change this and the above comment to something more obivous like the below:

`// ================== END ==================`